### DISCUSSION TITLE
Asyncio

### DISCUSSION THREAD

**Discussion by CiberNin** [r0x0r/pywebview#831] [Q&A] [answered]
So my backend is all asyncio, and the javascript end is all promises, but is there a way to connect the two asynchronously?
So if the function I pass to the api as a coroutine it will be called with _await_?

**Comment by r0x0r** [comment, marked as answer]
Currently no, async support should be added though. Here is a relevant ticket https://github.com/r0x0r/pywebview/issues/801
